lib/src/pub.dart

```diff
@@ -87,6 +108,19 @@ class Pubspec {
       this._contents,
       this._yamlMap);
 
+  static Pubspec init() {
+    var packageRoot = p.current;
+    var pubspecPath = p.join(packageRoot, _PUBSPEC);
+    var contents = "name: ${p.basename(packageRoot)}";
+    var yaml = loadYamlNode(contents, sourceUrl: pubspecPath);
+    var _pubspec = new Pubspec(pubspecPath, contents, yaml);
+
+    _pubspec.version = new Version.parse('0.1.0');
+
+    if(checkHasGitSync()) _pubspec.author = "${gitConfigUserNameSync()} <${gitConfigUserEmailSync()}>";
```

Comment:
Need some error handling here.  If git fails, or if the git config for either field was not set, it should not set the author.